infra/cloudflare-r2-wrangler/package.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "cloudflare-r2-wrangler",
```

WDYT of using

```suggestion
  "name": "@infra/cloudflare-r2-wrangler",
```
?

This will reduce the chance of clashes with existing npm packages.

---

Yep – https://github.com/blockprotocol/blockprotocol/pull/923/commits/62b1d290aca43f7fd7ade313a0538a5d7046009c